EIPS/eip-1.md

```diff
@@ -82,7 +82,7 @@ The following is the standardization process for all EIPs in all tracks:
 
 If this period results in necessary normative changes it will revert the EIP to `Review`.
 
-**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata in a backward compatible manner, or add non-normative clarifications.
```

Comment:
Personally I prefer the more permissive language we had before. With this new wording, the fix for EIP-712 wouldn't have been allowed.

Perhaps this is something we add to [EIP-5069](https://eips.ethereum.org/EIPS/eip-5069) instead?

Reply:
I personally concern this will break "FINAL"-ness. If we want to allow authors of an EIP have options to update, I am personally in favor of we change "LIVE" status to be allowed for ERC instead of changing requiurement of "FINAL"

Reply:
> With this new wording, the fix for EIP-712 wouldn't have been allowed.

With my definition, this would have been allowed. The sign typed data RPC was defined in such as to be unsatisfiable.

Reply:
The only normative changes should be errata, which make the document say what it should have said in the first place. Or in the (we hope) rare case that we need to make it say what was actually implemented, in spite of what it said.